Makefile

```
@@ -1,5 +1,5 @@
 .DEFAULT_GOAL := help
-COVERPKG := $(shell eval go list ./... | grep -v mocks | tr '\n' ',')
+COVERPKG := $(shell eval go list ./... | grep -v mocks | tr '\n' ',' | rev | cut -c 2- | rev)
```

```suggestion
COVERPKG := $(shell eval go list ./... | grep -v mocks |  paste -sd "," -)
```

This can be simplified to `go list ./... | grep -v mocks |  paste -sd "," -`

This avoids the issue of the two trailing characters in the first place.